data/network/src/main/java/io/filmtime/data/network/interceptor/ApiKeyInterceptor.kt

```diff
@@ -0,0 +1,24 @@
+package io.filmtime.data.network.interceptor
+
+import io.filmtime.data.network.BuildConfig
+import okhttp3.Interceptor
+import okhttp3.Response
+
+class ApiKeyInterceptor : Interceptor {
```

Comment:
Please rename it to `TmdbApiKeyInterceptor` since we might have other APIs that required this manipulation